spawns.lua

```diff
@@ -125,18 +125,65 @@ NorthGeorgiaFARPTransportSpawns = {
     ["SE"] = {Spawner("SE FARP HELO"),nil, SEFARPLogiSpawn},
     ["MK"] = {Spawner("MK FARP HELO"), nil, MaykopLogiSpawn}
 }
+
+-- Tells a tanker (or other aircraft for that matter) to orbit between two points at a set altitude
+
+BladderPos = {}
+BladderPos.hMaxFlightAlt	= 5486	-- meters [18 000']: Don't let the aircraft fly higher than this as Hogs won't be able to refuel. TODO: Make overrideable
+BladderPos.vSpeed			= 160		-- m/s [300kts]: Default speed of the unit
+
+function BladderPos.InitOnGroup(planeGroup, vSpeed)
+	local planeGroupName	= planeGroup:getName()
+	vSpeed = vSpeed or BladderPos.vSpeed
+	
+	local clouds = env.mission.weather.clouds
+	
+	-- Calculate orbit height. Ignore for partial cloud conditions
+	local hOrbit = BladderPos.hMaxFlightAlt
+	if (((clouds.base + clouds.thickness) >= hOrbit - 305) and (clouds.density >= 7)) then
+		hOrbit = math.min(clouds.base - 305, BladderPos.hMaxFlightAlt)
+	end
+	
+	local curRoute = mist.getGroupRoute(planeGroupName, true)
+	
+	for i = 1, #curRoute do
+		curRoute[i].alt = hOrbit
+		
+		-- Modify any orbit taskings
+		if #curRoute[i].task ~= nil then
```

Comment:
Oof. Maybe we can make this a single check, or another method? The indentation is getting pretty smelly here. 

I think lua does do short-circuiting, so you can do:
```lua
if a ~-nil and a.b ~= nil then
...
end
```
Since if the a is nil it'll not bother checking the rest, and not cause any nil indexing.

Reply:
Wilco, I had forgotten how to write that in.